app/src/main/java/org/thoughtcrime/securesms/home/HomeActivity.kt

```diff
@@ -331,7 +324,7 @@ class HomeActivity : PassphraseRequiredActionBarActivity(),
 
     private val GlobalSearchResult.contactAndGroupList: List<GlobalSearchAdapter.Model> get() =
         contacts.map { GlobalSearchAdapter.Model.Contact(it, it.nickname ?: it.name, it.accountID == publicKey) } +
-            threads.map(GlobalSearchAdapter.Model::GroupConversation)
+            threads.map(GlobalSearchAdapter.Model::LegacyGroupConversation)
```

Comment:
Does this need the new groupsV2 as well?

Reply:
The old `GropuConversation` is able to cater for new group so I've changed the naming